yarn.lock

```diff
@@ -5636,9 +5636,9 @@ typescript-formatter@4.0.1:
     editorconfig "^0.13.2"
     glob-expand "^0.2.1"
 
-typescript@2.6.1:
-  version "2.6.1"
-  resolved "https://registry.yarnpkg.com/typescript/-/typescript-2.6.1.tgz#ef39cdea27abac0b500242d6726ab90e0c846631"
+typescript@2.7.1:
```

Comment:
Why was this changed but `typescript-formatter` wasn't?

Reply:
Running `yarn` on my checkout of this branch causes many `yarn.lock` changes including `typescript-formatter`.

Reply:
Bad rebase. The error is from TS 2.7.2 with ts-formatter 7.1.0 though